src/dir.targets

```diff
@@ -10,6 +10,15 @@
   <Target Name="GetAssemblyVersion"
           Returns="$(AssemblyVersion)"/>
 
+  <!-- These gets overwritten when we restore the build tools and import the below targets -->
+  <Target Name="Build" Condition="!Exists('$(ToolsDir)Build.Common.targets')" >
+    <CallTarget Targets="_RestoreBuildTools" />
```

Comment:
I believe I understand what is happening here but I think we need a better comment here. Something that is mentioning that this Build will only ever run if build tools haven't been restored. It is probably even worth a message here (or perhaps the message already comes out of _RestoreBuildTools in this case), stating that build tools need to be restored.